server/tests/tasksTest.js

```diff
@@ -0,0 +1,31 @@
+import regeneratorRuntime from 'regenerator-runtime';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker'; // Fake data generator
+import app from '../app';
```

Comment:
Uncomment all those related to the  **dotenv**, because you are going to need them to access the token and other environmental variables.

Reply:
@fordarnold keep in mind that a pr should have one commit message.
Currently, this pr has 9 commits check it out.